data/incipias/incipias ships.txt

```diff
@@ -0,0 +1,239 @@
+	description "Although the Nimbus is labeled as a medium warship, it is multipurpose; whether one uses it for mining, passenger transport or cargo, it will perform its job nicely. It is only occasionally used in combat because the Incipias are not currently fighting open wars."
+
+ship "Nimbus" "Nimbus (Civil)"
+	outfits
+		"Liquid-Class Shield"
+		"Metallic Hydrogen Cell"
+
+		"MH Red-Class Engine"
+		"MH Red-Class Steering"
+
+ship "Nimbus" "Nimbus (Mining)"
+	outfits
+		"Liquid-Class Shield"
+		"Metallic Hydrogen Cell"
+		"Plasma Discharger" 4
+		"Stagnation Beam" # to be replaced with a flotsam tractor beam
+		"Cloud Piercer" 20
+
+		"MH Red-Class Engine"
+		"MH Red-Class Steering"
+
+ship "Nimbo Stratus"
+	"plural" "Strati"
+	"sprite" ship/stratus
+	"thumbnail" thumbnail/stratus
+	"swizzle" 0
+	attributes
+		"category" "Utility"
+		"cost" 50000
+		"hull" 500
+		"shields" 700
+		"required crew" 5
+		"bunks" 10
+		"mass" 534
+		"drag" 8.4
+		"heat dissipation" .67
+		"cargo space" 50
+		"outfit space" 270
+		"weapon capacity" 50
+		"engine capacity" 60
+		"gaslining" 1
+		"fuel capacity" 200
+
+	outfits
+		"Gas-Class Shield"
+		"Metallic Hydrogen Cell"
+		"Plasma Discharger" 2
+
+		"MH Blue-Class Engine"
+		"MH Blue-Class Steering"
+
+	gun -13 -48.5
+	gun 13 -48.5
+	engine 0 61.5
+	explode "medium explosion" 10
+	explode "large explosion" 20
+	"final explode" "final explosion medium"
+	description "Mining has always been a big business in Incipias society, which is why the Stratus is the most common ship for beginning captains. Its cheap price tag and ability to bring money in quickly ensure its continuing popularity. While the Stratus could theoretically be used in combat, it stands no chance against the Nimbus and therefore has never been used by the Hicemus military forces."
+
+ship "Nimbo Cirrus"
+	"plural" "Cirri"
+	"sprite" ship/cirrus
+	"thumbnail" thumbnail/cirrus
+	"swizzle" 0
+	attributes
+		"category" "Light Freighter"
+		"cost" 100000
+		"hull" 750
+		"shields" 1500
+		"required crew" 7
+		"bunks" 14
+		"mass" 800
+		"drag" 8.7
+		"heat dissipation" .6
+		"cargo space" 200
+		"outfit space" 160
+		"engine capacity" 60
+		"gaslining" 1
+		"fuel capacity" 200
+	outfits
+		"Gas-Class Shield"
+		"Metallic Hydrogen Cell"
+
+		"MH Green-Class Engine"
+		"MH Green-Class Steering"
+
+	engine -32 80
+	engine 32 80
+	engine 0 80
+	explode "medium explosion" 20
+	explode "large explosion" 30
+	"final explode" "final explosion medium"
+	description "When Pon'tes was built, the need arose for a ship that could carry large amounts of material and supplies into orbit. When the Nimbus proved insufficient at this task, the Cirrus, a specialized cargo ship, was built. The inhabitants of Pon'tes rely on its regular deliveries for basic goods and equipment."
+
+ship "Gero"
+	"sprite" ship/gero
+	"thumbnail" thumbnail/gero
+	"swizzle" 0
+	attributes
+		"category" "Transport"
+		"cost" 100000
```

Comment:
This probably ought to be a little under a million. 72 bunks is quite a lot, even if that's largely the only thing this ship has going for it.

Reply:
```suggestion
		"cost" 985000
```